paf-mvp-pattern-library-audit/stories/components/Icons.js

```diff
@@ -0,0 +1,99 @@
+export const User = () => `<svg width="1em" height="1em" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Would it be possible to store all these icons in separate svg files that get bundled at build time?
This way the icons would be "displayable" in any SVG editor and even previewed by GitHub.